src/fft/polynomial.rs

```diff
@@ -7,8 +7,11 @@
 //! This module contains an implementation of a polynomial in coefficient form
 //! Where each coefficient is represented using a position in the underlying vector.
 use super::{EvaluationDomain, Evaluations};
+use crate::error::Error;
 use crate::util;
 use dusk_bls12_381::BlsScalar;
+use dusk_bytes::{DeserializableSlice, Serializable};
+#[cfg(test)]
 use rand_core::{CryptoRng, RngCore};
```

Comment:
Can't this moved directly in the `mod test`?

Reply:
Yes, good catch!!